### PR TITLE
doc/tools/libzbc.md - refine libzbc.md

### PR DESCRIPTION
**Pull request by zdover23**
This PR refines the English in the libzbc.md file.
It contains one TODO, which awaits Damien's judgment.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

### CODE REVIEW

File: docs/tools/libzbc.md [hunk 1]
```diff
@@ -31,35 +31,34 @@ the <a href="https://github.com/westerndigitalcorporation/"
...
+in more detail [here](../tests/zbc-tests.md).
 
 ## Overview
 
 *libzbc* provides a unified application programming interface (API) that is
-independent of the zone model and interface of the disk being used. Internally,
-four different types of device drivers are used to handle device interface
-dependent commands.
+independent of the zone model and the interface of the disk being used.
+Internally, four different types of device drivers are used to handle commands that are dependent on the device interface:
```
Comment (damien-lemoal) [inline docs/tools/libzbc.md:44]: long line...

File: docs/tools/libzbc.md [hunk 2]
```diff
@@ -31,35 +31,34 @@ the <a href="https://github.com/westerndigitalcorporation/"
...
 
-* **ZAC ATA Driver** This driver is used to handle direct delivery of ATA
+* **ZAC ATA Driver** This driver is used to handle the direct delivery of ATA
   commands to ZAC disks through the SCSI generic driver (direct device
   access interface).
 * **ZBC SCSI Driver** This driver primarily handles SCSI commands directed at
   ZBC SCSI disks, but can also be used to control ZAC ATA disks if a functional
   SCSI to ATA (SAT) command translation layer is functional in the command path.
```
Comment (damien-lemoal) [inline docs/tools/libzbc.md:51]: Repeating functional here. We could improve this with something like:
...if a functional SCSI to ATA (SAT) command translation layer exists in the command path.

File: docs/tools/libzbc.md [hunk 3]
```diff
@@ -145,18 +146,19 @@ development and tests.
...
-inability to report detailed error information when the zoned block device
-driver is used. The reason for this is that the kernel I/O stack does not have
-the ability to propagate up to the application the detailed information provided
-in command sense data with failed commands.
+All functions behave in the same manner regardless of the type of disk that is
+used. The only exception to this is the `zbc_errno()` function's inability to
+report detailed error information when the zoned block device driver is used.
+The reason for this is that the kernel I/O stack does not have the ability to
+propagate up to the application the detailed information that is provided in
+command sense data [TODO: Damien, what is "command sense data"?] with failed
```
Comment (damien-lemoal) [inline docs/tools/libzbc.md:154]: SCSI and ATA devices report errors details using "sense data". This is a well-known term for SCSI/ATA devices. So maybe change this to "...that is provided in SCSI/ATA sense data for failed commands."

File: docs/tools/libzbc.md [hunk 4]
```diff
@@ -258,9 +260,9 @@ Zone 55878: type 0x2 (Sequential-write-required), cond 0x1 (Empty), reset recomm
 Zone 55879: type 0x2 (Sequential-write-required), cond 0x1 (Empty), reset recommended 0, non_seq 0, sector 29296689152, 524288 sectors, wp 29296689152
 ```
 
-The output of *zbc_report_zones* can also be limited using the option `-ro`
-(reporting options) for filtering the zones reported, or the option `-n` to
-limit the output to the number of zones that would be reported.
+The output of *zbc_report_zones* can be limited by using the option `-ro`
+(reporting options) to filter which zones are reported, or by using the option
+`-n` to limit the output to the number of zones reported.
```
Comment (damien-lemoal) [inline docs/tools/libzbc.md:265]: Bug: this should be: "...to limit the output to the number of zones specified by this option."

File: docs/tools/libzbc.md (hunk 3 above)
Comment (zdover23) [reply docs/tools/libzbc.md:154]: <montgomery burns>Excellent.</montgomery burns>

File: docs/tools/libzbc.md (hunk 4 above)
Comment (zdover23) [reply docs/tools/libzbc.md:265]: I've gone slightly more pedantic:

> ...to limit the output to the number of zones specified as an argument of
> this option.

**Comment by damien-lemoal** [comment]
Applied. Thanks !

**Pull request merged**